modules/ROOT/pages/6.4.1-release-notes.adoc

```diff
@@ -0,0 +1,1117 @@
+
+=== The colour picker reset to default when formatting pre element with addtional format options
+//#TINY-9342
+
+In previous versions of {productname}, formatting elements with a `pre-formatted` tag, and applying a background color from the color swatch would cause formatting issues when additional formatting options such as bold and italics were applied to the `pre-formatted` element. When these formatting conditions were combined, the color picker would reset the users defined color value, which would trigger the color picker to reset this value to default "000000".
+
+In {productname} 6.4.1, this issue has been resolved. Any pre-formatted elements what have additional formatting such as a color from the color picker no longer reset to its default color.
+
+=== Editor increases the height indefinitely when using autoresize with content_css document
+//#TINY-8872
+
+In previous versions of {productname}, It was discovered that the editor had some issues with the autoresize function when using content_css documents, which caused the editor to have a indefinite height when content was being added.
+
+To fix this issue, {productname} 6.4.1 fixed this issue by addressing the conflict between the document skin's CSS and the autoresize bottom margin setting. After this fix, the resize function now checks for loops and considers the margin bottom as always set to 0 instead of autoresize bottom margin, which ensutes that the editor's height is calculated correctly. 
+
+=== Plugin - Search and Replace - Throws a JS console error if you search for a string that doesn't exist and try to click on the preferences button.
+//#TINY-9686
+
+In previous versions of {productname}, the 'Search and Replace' plugin had a known issue that caused a console error when searching for a string that wasn't in the editor content, and then attempting to click on the `preferences` button.
+
+As a result, an "incorrect type" error would prevent users from opening the preferences section.
+
+In {productname} 6.4.1, this issue has been resolved.
+
+=== Toolbar doesn't hide when editor is out of view in scrollable container
+//#TINY-9652
+
+In previous versions of {productname}, when the editor is within a scrollable container, the toolbar would remain visible when the editor is scrolled out of view.
+
+With the release of {productname} 6.4.1, the new `ui_mode` option provides improvements to editors within scrollable contexts, including hiding the toolbar when the editor is out of view when the `ui_mode` option is set to 'split'.
+
+=== `color_cols`: 0 caused the page to crash
+//#TINY-9649
+
+In previous versions of {productname}, the color swatch would attempt to display 0 columns of colors when the relevant `color_cols`, `color_cols_foreground` or `color_cols_background` options were set to 0. As a consequence, the tab would crash when opening that color swatch.
```

Comment:
It was fixed in 6.3, so I figured we should remove the 6.4.1 mention...

```suggestion
=== Editor increases the height indefinitely when using autoresize with `content_css: 'document'`
//#TINY-8872

In previous versions of {productname}, it was discovered that the editor had some issues with the autoresize plugin when using `content_css: 'document'`, which caused the editor height to expand indefinitely.

To fix this issue, {productname} addressed the conflict between the document skin's CSS and the autoresize bottom margin setting. After this fix, the resize function now checks for loops and considers the margin bottom as always set to 0 instead of autoresize bottom margin, which ensutes that the editor's height is calculated correctly. 
```